docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc

```diff
@@ -0,0 +1 @@
+Please note that Elastic’s global managed testing infrastructure is a charged for service, as shown on the https://www.elastic.co/pricing[pricing page].
```

Comment:
The phrase "is a charged for service" sounds strange to me. What about something like this? Not sure if it should be "when you run monitors" or "when you create monitors":

```suggestion
When you run monitors on Elastic’s global managed testing infrastructure, you will be charged as outlined on the https://www.elastic.co/pricing[pricing page].
```

Reply:
Thanks @colleenmcginnis that's much better than what I had written as I wasn't sure what @drewpost wanted to include.

@drewpost are you happy to go with this, or something else?

Reply:
How about. "Executing synthetic tests on the Elastic global managed testing infrastructure incurs an additional charge. Tests are charged under one of two new billing dimensions depending on the type of test. Browser tests are charged for on a per test run basis and lightweight ping tests have an all you can eat model charged by region. Please refer to https://www.elastic.co/pricing for full details and current pricing" @paulb-elastic @colleenmcginnis